langlib/lang.test/src/main/java/org/ballerinalang/langlib/test/AssertError.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.ballerinalang.langlib.test;
+
+import org.ballerinalang.jvm.BallerinaErrors;
+import org.ballerinalang.jvm.scheduling.Strand;
+import org.ballerinalang.jvm.values.ErrorValue;
+import org.ballerinalang.model.types.TypeKind;
+import org.ballerinalang.natives.annotations.Argument;
+import org.ballerinalang.natives.annotations.BallerinaFunction;
+
+/**
+ * Native implementation of assertError(anydata|error value).
+ *
+ * @since 1.3.0
+ */
+@BallerinaFunction(
+        orgName = "ballerina", packageName = "lang.test", functionName = "assertError",
+        args = {@Argument(name = "value", type = TypeKind.UNION)},
+        isPublic = true
+)
+public class AssertError {
+    public static void assertError(Strand strand, Object value) {
+        if (!(value instanceof ErrorValue)) {
+            throw BallerinaErrors.createError("Not an Error");
+        }
```

Comment:
We don't usually use `instanceof` checks. Shall we do something like

```suggestion
        if (TypeChecker.getType(objVal).getTag() != TypeTags.ERROR) {
            throw BallerinaErrors.createError(ERR_REASON);
        }
```